docs/pages/index.mdx

```diff
@@ -114,83 +106,91 @@ Use the SDK for everything, or pick and choose the pieces you want to use. The S
     style={{ textAlign: "left", alignItems: "flex-start" }}
     href="/wallets/core/overview"
   >
-    Svelte, Vue, etc.
+    Framework-agnostic implementation for any web application.
   </Card>
   <Card
-    title="Infra"
+    title="APIs"
     icon="wrench"
     iconPosition="top"
     style={{ textAlign: "left", alignItems: "flex-start" }}
-    href="/wallets/transactions/low-level-infra/overview"
+    href="/wallets/reference/smart-wallet-quickstart"
   >
-    Integrate with Alchemy's infra
+    Server-side wallet management for high-volume applications.
   </Card>
+</CardGroup>
+
+## Common starting places
+
+<CardGroup cols={2}>
   <Card
-    title="Smart Contracts"
-    icon="file-contract"
+    title="Build a new app"
+    icon="fa-brands fa-react"
     iconPosition="top"
     style={{ textAlign: "left", alignItems: "flex-start" }}
-    href="/wallets/smart-contracts/choosing-a-smart-account"
+    href="/wallets/react/overview"
   >
-    Use Alchemy's Smart Contracts
+    Build a complete onchain app from scratch with embedded wallets and
+    transactions.
   </Card>
   <Card
-    title="Signer"
-    icon="pen-clip"
+    title="Upgrade existing wallets"
+    icon="fa-brands fa-react"
     iconPosition="top"
     style={{ textAlign: "left", alignItems: "flex-start" }}
-    href="/wallets/signer/what-is-a-signer"
+    href="/wallets/react-native/overview"
   >
-    Use Alchemy's Signer
+    Upgrade to smart wallets using EIP-7702 or direct wagmi integration.
   </Card>
   <Card
-    title="EIP-7702"
-    icon="diamond"
+    title="Bring your app onchain"
+    icon="ellipsis"
     iconPosition="top"
     style={{ textAlign: "left", alignItems: "flex-start" }}
-    href="/wallets/react/using-7702"
+    href="/wallets/core/overview"
   >
-    Upgrade EOAs to smart accounts
+    Add wallet and transaction functionality to existing web2 applications.
+  </Card>
+  <Card
+    title="Manage wallets from your backend"
+    icon="wrench"
+    iconPosition="top"
+    style={{ textAlign: "left", alignItems: "flex-start" }}
+    href="/wallets/reference/smart-wallet-quickstart"
+  >
+    Server side applications with signing and sending on your backend.
```

Comment:
(Nit)
```suggestion
    Server-side applications with signing and sending on your backend.
```
because you wrote it that way in the previous section